matrix-sdk/src/main/java/org/matrix/androidsdk/crypto/verification/VerificationEmoji.kt

```diff
@@ -0,0 +1,212 @@
+        when (code) {
+            0 -> return EmojiRepresentation("🐶", context?.getString(R.string.verification_emoji_dog)
+                    ?: "")
+            1 -> return EmojiRepresentation("🐱", context?.getString(R.string.verification_emoji_cat)
+                    ?: "")
+            2 -> return EmojiRepresentation("🦁", context?.getString(R.string.verification_emoji_lion)
+                    ?: "")
+            3 -> return EmojiRepresentation("🐎", context?.getString(R.string.verification_emoji_horse)
+                    ?: "")
+            4 -> return EmojiRepresentation("🦄", context?.getString(R.string.verification_emoji_unicorn)
+                    ?: "")
+            5 -> return EmojiRepresentation("🐷", context?.getString(R.string.verification_emoji_pig)
+                    ?: "")
+            6 -> return EmojiRepresentation("🐘", context?.getString(R.string.verification_emoji_elephant)
+                    ?: "")
+            7 -> return EmojiRepresentation("🐰", context?.getString(R.string.verification_emoji_rabbit)
+                    ?: "")
+            8 -> return EmojiRepresentation("🐼", context?.getString(R.string.verification_emoji_panda)
+                    ?: "")
+            9 -> return EmojiRepresentation("🐓", context?.getString(R.string.verification_emoji_rooster)
+                    ?: "")
+            10 -> return EmojiRepresentation("🐧", context?.getString(R.string.verification_emoji_penguin)
+                    ?: "")
+            11 -> return EmojiRepresentation("🐢", context?.getString(R.string.verification_emoji_turtle)
+                    ?: "")
+            12 -> return EmojiRepresentation("🐟", context?.getString(R.string.verification_emoji_fish)
+                    ?: "")
+            13 -> return EmojiRepresentation("🐙", context?.getString(R.string.verification_emoji_octopus)
+                    ?: "")
+            14 -> return EmojiRepresentation("🦋", context?.getString(R.string.verification_emoji_butterfly)
+                    ?: "")
+            15 -> return EmojiRepresentation("🌷", context?.getString(R.string.verification_emoji_flower)
+                    ?: "")
+            16 -> return EmojiRepresentation("🌳", context?.getString(R.string.verification_emoji_tree)
+                    ?: "")
+            17 -> return EmojiRepresentation("🌵", context?.getString(R.string.verification_emoji_cactus)
+                    ?: "")
+            18 -> return EmojiRepresentation("🍄", context?.getString(R.string.verification_emoji_mushroom)
+                    ?: "")
+            19 -> return EmojiRepresentation("🌏", context?.getString(R.string.verification_emoji_globe)
+                    ?: "")
+            20 -> return EmojiRepresentation("🌙", context?.getString(R.string.verification_emoji_moon)
+                    ?: "")
+            21 -> return EmojiRepresentation("☁", context?.getString(R.string.verification_emoji_cloud)
```

Comment:
The spec listed some uncolored emojis and those sadly got copied here. The js sdk replaced the uncolored ones but sadly the spec still didn't get updated.

There are 8 uncolored emojis: cloud, heart, smiley, umbrella, pencil, scissors, telephone, airplane.

The correct symbols are: ☁️  ❤️  😀 ☂️  ✏️  ✂️  ☎️  ✈️. Or you can view them in the updated spec as well.

The js sdk commit fixing this can be found here: https://github.com/matrix-org/matrix-js-sdk/commit/71a01ec23479b1298ad009fe6cd93609b1058bc8#diff-29e9a46fcc7e468e1fbf6b2534910852.

Reply:
Thanks, poljar. I will have to test on old android device to see how colored emoji are displayed.
(The updated spec? :D)

Reply:
What about the updated spec? It has been updated a couple of days ago and is found on the usual place [here](https://docs.google.com/document/d/1SXmyjyNqClJ5bTHtwvp8tT1Db4pjlGVxfPQNdlQILqU/).

Reply:
thx poljar